frontend/frontendAngular/src/app/components/navbar/navbar.component.html

```diff
@@ -1,7 +1,7 @@
 <p>
   <mat-toolbar color="primary">
     <span><a class="logo" (click)="direct()">Mark.it</a></span>
-    <ng-container *ngIf="!((localStorage.get('token'))==null) && this.localStorage.get('isCompany')">
```

Comment:
substituir o get por decodePayloadjwt(this.localStorage.get('token')).cnpj == null"